contentful/apps
# feat: add initial Microsoft Teams app [INTEG-1468]

## Purpose

This PR creates an initial app for Microsoft Teams.

## Approach

Utilized Create Contentful App using the Vite configuration. Also made some additional updates:

- Update test scripts
- Add linting
- Add support for absolute paths
- Add `.nvmrc`

## Testing steps

Ran app locally and was able to verify that the app rendered in our Microsoft Teams development environment. Also ran `test`, `lint`, and `build` scripts.

## Breaking Changes

## Dependencies and/or References

## Deployment

Deploy scripts will be added in a separate PR.
